vercel.json

```diff
@@ -4,7 +4,7 @@
     "silent": true,
     "autoJobCancelation": true
   },
-  "installCommand": "yarn setup",
+  "installCommand": "yarn install --immutable",
```

Comment:
Why does this need to be changed?

Reply:
I need to rebase the target branch, this comes from `main` because on `main` we don't have setup no frozen-lockfile.